crates/sargon/src/types/samples/identity_address_samples.rs

```diff
@@ -0,0 +1,23 @@
+use crate::prelude::*;
+
+pub(crate) static ALL_IDENTITY_ADDRESSES_SAMPLES: Lazy<[IdentityAddress; 10]> =
+    Lazy::new(|| {
+        [
+            IdentityAddress::random(NetworkID::Mainnet),
```

Comment:
same question about index `0` and `1` as for Account: https://github.com/radixdlt/sargon/pull/235/files#r1798836082